src/configure/configure.ts

```diff
@@ -137,19 +143,21 @@ class Orchestrator {
         const azureResourceSelector = ResourceSelectorFactory.getAzureResourceSelector(targetType);
         this.inputs.targetResource.resource = await azureResourceSelector.getAzureResource(this.inputs);
         this.azureResourceClient = ResourceSelectorFactory.getAzureResourceClient(targetType, this.inputs.azureSession.credentials,
-            this.inputs.azureSession.environment, this.inputs.azureSession.tenantId, this.inputs.subscriptionId);
+        this.inputs.azureSession.environment, this.inputs.azureSession.tenantId, this.inputs.subscriptionId);
         telemetryHelper.setTelemetry(TelemetryKeys.resourceType, this.inputs.targetResource.resource.type);
-        telemetryHelper.setTelemetry(TelemetryKeys.resourceKind, this.inputs.targetResource.resource.kind);
-        telemetryHelper.setTelemetry(TelemetryKeys.resourceIdHash, Utilities.createSha256Hash(this.inputs.targetResource.resource.id));
+        if ( targetType === TargetResourceType.WebApp) {
+            telemetryHelper.setTelemetry(TelemetryKeys.resourceKind, this.inputs.targetResource.resource.kind);
+            telemetryHelper.setTelemetry(TelemetryKeys.resourceIdHash, Utilities.createSha256Hash(this.inputs.targetResource.resource.id));
```

Comment:
Are we logging telemetry like this in case of remote provisioning? e.g. resourceId hash

Reply:
This seems a bug to me for AKS case, because for AKS these values will be undefined  at this point of function this.inputs.targetResource.resource.kind,
this.inputs.targetResource.resource.id.

Reply:
Okay, got it. You mean we need to log hash of aks cluster id as well. Currently we are not doing. I will add it

Reply:
Will change this as a part of template service template to make it generic. there we have params logtelemetry and hash. that we can use

Reply:
makes sense